apps/proxy-auth/src/app/otp/send-otp/send-otp.component.ts

```diff
@@ -19,6 +19,7 @@ import {
 import { FeatureServiceIds } from '@proxy/models/features-model';
 import { OtpWidgetService } from '../service/otp-widget.service';
 import { HttpErrorResponse } from '@angular/common/http';
+import { F } from '@angular/cdk/keycodes';
```

Comment:
why dont you check these useless imports before giving PR
Please review your PR thoroughly before asking for review